<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>battleOrderDialog</class>
 <widget class="QDialog" name="battleOrderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditCount"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelName">
     <property name="text">
      <string>Наименование</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelCount">
     <property name="text">
      <string>Количество</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelPlace">
     <property name="text">
      <string>Место хранения</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditPlace"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
